src/_data/side-nav.yml

```diff
@@ -242,6 +232,17 @@
     - title: Environment declarations
       permalink: /guides/environment-declarations
 
+- title: Interoperability
+  expanded: false
+  children:
+  - title: JavaScript interop
+    permalink: /interop/js-interop
```

Comment:
Actually, consider moving this to the bottom of the list for now, especially while it's in limbo.